lib/src/test/java/com/auth0/jwt/JWTCreatorTest.java

```diff
@@ -474,18 +510,20 @@ public void shouldAcceptCustomMapClaimOfBasicObjectTypes() throws Exception {
         data.put("long", Long.MAX_VALUE);
         data.put("double", 123.456d);
         data.put("date", new Date(123L));
+        data.put("instant", Instant.ofEpochSecond(123L));
         data.put("boolean", true);
 
         // array types
         data.put("intArray", new Integer[]{3, 5});
         data.put("longArray", new Long[]{Long.MAX_VALUE, Long.MIN_VALUE});
         data.put("stringArray", new String[]{"string"});
 
-        data.put("list", Arrays.asList("a", "b", "c"));
+        data.put("list", Arrays.asList("a", "b", "c", Instant.ofEpochSecond(41L)));
```

Comment:
I don't think this is correct. Shouldn't arrays/lists have the same type across items? 🤔 If you need to test serialization of Instants within lists/maps, do that on a separate claim.

Reply:
As currently implemented, they do not need to have the same type. For example, this test is setting up an array with each item a different basic type: https://github.com/auth0/java-jwt/blob/master/lib/src/test/java/com/auth0/jwt/JWTCreatorTest.java#L524